chore: update README for auto subscribe (#10355)
emqx/emqx: apps/emqx_auto_subscribe/README.md

# Auto Subscribe

> This application can help clients automatically subscribe to topics compiled from user definitions when they connect, and the clients no longer need to send the MQTT `SUBSCRIBE ` request.

Reviewer: ```suggestion
This application automatically subscribes clients to topics compiled from user definitions when they connect, so the clients no longer need to send the MQTT `SUBSCRIBE ` request.
```
